docs/community/tutorials.md

```diff
@@ -0,0 +1,12 @@
+# Tutorials
+
+This page lists further learning materials and code walkthroughs for the H3 library and bindings.
+
+## JavaScript
+
+- [H3 Tutorials](https://beta.observablehq.com/collection/@nrabinowitz/h3-tutorial)
```

Comment:
Maybe expand the descriptions here?

- H3 Tutorials on Observable
- Usage (iPython Notebook)
- Unified Data Layers (iPython Notebook)